src/main.dev.ts

```diff
@@ -140,7 +140,16 @@ app.on('window-all-closed', () => {
   }
 });
 
-app.whenReady().then(createWindow).catch(console.log);
+app
+  .whenReady()
+  .then(() => {
+    globalShortcut.register('CommandOrControl+O', () => {
+      mainWindow.show();
+    });
+    return mainWindow;
+  })
+  .then(createWindow)
```

Comment:
`createWindow` instantiates the mainWindow object. Prior to this line, it should not exist so I'm not sure the code above will perform

Reply:
The reference for it exists prior and gets loaded from that async function for use later, but would be safer to register it after the full window is created.